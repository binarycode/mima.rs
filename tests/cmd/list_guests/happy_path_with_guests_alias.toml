args = 'guests'
status.code = 0
