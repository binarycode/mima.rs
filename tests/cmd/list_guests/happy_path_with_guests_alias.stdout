ID    BOOTED  SPICE  DESCRIPTION
beta  false   5901   foo
zero  true    5902   bar
